flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -1629,9 +1630,9 @@ private void closeLocalRegistry() {
 
 		private boolean isWithLocalRecovery(
 			CheckpointOptions.CheckpointType checkpointType,
-			RocksDBStateBackend.LocalRecoveryMode recoveryMode) {
+			LocalRecoveryConfig.LocalRecoveryMode recoveryMode) {
 			// we use local recovery when it is activated and we are not taking a savepoint.
-			return RocksDBStateBackend.LocalRecoveryMode.ENABLE_FILE_BASED.equals(recoveryMode)
+			return LocalRecoveryConfig.LocalRecoveryMode.ENABLE_FILE_BASED.equals(recoveryMode)
```

Comment:
I prefer for enum comparisons `==`, but we had this discussion before ;-)

Reply:
👍